config/config.go

```diff
@@ -0,0 +1,65 @@
+package config
```

Comment:
comment - always wondered but should we call this package envvars since it's all about that?